frontend/src/components/ListContainer/ListContainer.jsx

```diff
@@ -171,14 +171,15 @@ export default function ListContainer({
                 </ImageListItem>
               ) : null}
               {assignedScenarios && assignedScenarios.length > 0
-                ? assignedScenarios.map(({ _id, name }) => (
+                ? assignedScenarios.map((item) => (
```

Comment:
nit: `scenario` is better name?

Reply:
ah - the tricky one here being that `item` can be a scenario or a scene.

The whole `ListComponent` shouldn't be the way it is... Ideally the contents are abstracted away, but alas time is our limiting factor here.